src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellSearchView.cs

```diff
@@ -317,12 +317,22 @@ AImageButton CreateImageButton(Context context, BindableObject bindable, Bindabl
 			result.SetScaleType(ImageView.ScaleType.FitCenter);
 
 			if (bindable.GetValue(property) is ImageSource image)
```

Comment:
Could we add a related UITest? Let me know if you need help with this or if you want me to add it myself.

Reply:
Done. I hope. Is this UITest what you had in mind?

Reply:
@jsuarezruiz I added a uitest that verifies the default icons and one that verifies the custom icons.